coops-ui/src/views/StandaloneDirectorsFiling.vue

```diff
@@ -181,7 +182,14 @@
                 <header>
                   <h1 id="filing-header-review">Review: Director Change </h1>
                 </header>
-
+                <section v-if="complianceDialogMsg">
+                  <v-alert type="info" outlined
+                  icon="mdi-information"
+                  class="white-background"
+                  >
+                    <p class="complianceDialogMsg">{{ complianceDialogMsg.msg }}</p>
+                </v-alert>
+                </section>
```

Comment:
Something isn't right here: `</v-alert>` and `</section>` shouldn't be at the same indent level. Please fix.